config/application.yml.default

```diff
@@ -229,6 +229,9 @@ participate_in_dap: false
 password_max_attempts: 3
 personal_key_retired: true
 phone_carrier_registration_blocklist_array: '[]'
+short_term_phone_otp_max_attempt_window_in_seconds: 10
+short_term_phone_otp_max_attempts: 2
```

Comment:
My initial impression was that this would allow someone to request 2 codes within 10 seconds, but I can only request 1 code within 10 seconds. I think either would be fine, but the configuration value felt a little misleading to me (or at least unexpected), i.e. I would expect this value to be `1` if I can only request 1 code every 10 seconds.

Reply:
Yeah, the way the RateLimiter class is kind of weird in that way unfortunately :/

We could switch https://github.com/18F/identity-idp/blob/4d41f77161e35a91a62a039de390dfb70d40bcec/app/services/rate_limiter.rb#L67 to be `>` rather than `>=` and reduce everything by one 😬